apps/judicial-system/web/messages/index.ts

```diff
@@ -43,7 +43,7 @@ export * from './Core/sections'
 export * from './Core/caseDates'
 export * from './Core/caseFilesAccordion'
 export * from './Core/commentsAccordion'
-export * from './Core/courtRecordAccordion'
+export * from '../src/components/AccordionItems/CourtRecordAccordionItem/strings.courtRecordAccordion'
```

Comment:
You will need to change the `extract-strings` script in `apps/judicial-system/web/project.json` so that these strings are included.